### DISCUSSION TITLE
What is the odometry method

### DISCUSSION THREAD

**Discussion by SiinaIng** [introlab/rtabmap#1004] [Q&A]
Hello @matlabbe 

What odometry is this launch configuration trying to use? I know it's ICP, but is it from Lidar or depth scans?  

`roslaunch rtabmap_ros rtabmap.launch stereo:=false \
   rgb_topic:=/zed2/zed_node/rgb/image_rect_color \
   camera_info_topic:=/zed2/zed_node/rgb/camera_info use_sim_time:=true \
   depth_topic:=/zed2/zed_node/depth/depth_registered \
   depth_camera_info_topic:=/zed2/zed_node/depth/camera_info \
   frame_id:=base_link \
   subscribe_scan_cloud:=true \
   rgbd_sync:=true \
   scan_cloud_topic:=/os_cloud_node/points \
   scan_cloud_max_points:=131072  \
   approx_sync:=true \
   icp_odometry:=true \
   args:="-d  \
      --Odom/ResetCountdown 0 \
      --delete_db_on_start \
      --RGBD/CreateOccupancyGrid true \
      --Rtabmap/DetectionRate 0.5 \
      --Odom/ScanKeyFrameThr 0.8 \
      --OdomF2M/ScanMaxSize 10000  \
      --OdomF2M/ScanSubtractRadius 0.5  \
      --Icp/PM true \
      --Icp/VoxelSize 0.05   \
      --Icp/MaxTranslation 0   \
      --Icp/MaxCorrespondenceDistance 0.5 \
      --Icp/PMOutlierRatio 0.85 \
      --Icp/Iterations 10 \
      --Icp/PointToPlane false \
      --Icp/PMMatcherKnn 3 \
      --Icp/PMMatcherEpsilon 1 \
      --Icp/Epsilon 0.0001 \
      --Icp/PointToPlaneK 0 \
      --Icp/PointToPlaneRadius 0 \
      --Icp/CorrespondenceRatio 0.01"`

**Comment by matlabbe** [comment]
Duplicate of https://github.com/introlab/rtabmap_ros/issues/933
